Gems/ROS2/Code/Source/Frame/ROS2FrameComponent.cpp

```diff
@@ -73,6 +73,29 @@ namespace ROS2
                 GetFrameID().data(),
                 IsDynamic() ? "continuously to /tf" : "once to /tf_static");
 
+            if (!IsDynamic())
+            {
+                bool hasMovableJoints = false;
+                const auto& components = this->m_entity->GetComponents();
+                for (const auto component : components)
+                {
+                    const auto type = component->GetUnderlyingComponentType();
+
+                    if (type == AZ::Uuid("{9C5B955C-6C80-45FA-855D-DDA449C85313}") || // PhysXHingeJoint
```

Comment:
Would it be easier if we considered any ROS2Frame without, specifically, a FixedJoint to be a dynamic frame?
It does look correct and easier to maintain (consider new joint types being added).